exp/production.md

```diff
@@ -0,0 +1,84 @@
+When you are planning a production deployment, you should consider the aspects that you want to customize for best results. This topic covers the most common and important requirements when you are planning to move your Anbox Cloud deployment to production. While there could be other specifically tailored needs, this topic helps you identify your basic requirements and deployment strategy.
+
+## Hardware requirements 
+
+While you can run Anbox Cloud on a single machine, you should use several machines for a production environment. Depending on your workload and the type of deployment you choose, hardware requirements can differ. See [requirements](https://anbox-cloud.io/docs/requirements) to get an idea about the kind of hardware that is required to run a full Anbox Cloud deployment with the streaming stack or a minimal core version of the deployment without the streaming stack. 
+
+You should pay special attention to the hardware requirements of applications that are used but not maintained by Anbox Cloud. Anbox Cloud Appliance has a different set of hardware requirements but we do not recommend the appliance for a production environment.
+
+## Software requirements 
+
+Anbox Cloud deployments require Ubuntu operating system. You should consider the Ubuntu OS, LXD, and Juju versions supported by Anbox Cloud for a production deployment. See [requirements](https://anbox-cloud.io/docs/requirements) to see compatible software versions.
+
+## Networking requirements 
+
+Consider the following questions to decide your networking requirements:
+
+* In your deployment, how many virtual private clouds(VPC) do you need per cluster?
+* Do you need subnets? If yes, what is the range of the subnets that you want to define?
+* Have you defined your AWS regions?
+* For each cluster, how do you want to map the subnets within the virtual private clouds into Juju spaces?
+* Where is your AMS Application Registry (AAR) deployed? How will the AMS in other clusters connect to AAR? Define your VPC peering requirements based on the number of clusters you have and where AAR is deployed.
+
+## Storage requirements
+Based on your safety protocols, think about where you would want to host your storage.
+
+## Scalability, High availability, and redundancy
+
+With Anbox Cloud, you can scale your deployment to as many clusters and subclusters as you require. It is a good idea to define how many Anbox clusters you want to deploy and how many subclusters per cluster. 
+
+You should also define the number of instances that you require for each model/service. For example, think about the number of Juju controller instances that you need.
```

Comment:
Isn't this more about finding out the number of instances you can put on a machine instead of defining it? For instance, a given GPU has a limited capacity, so this already gives an upper limit. Or is it more about how many sessions you are ready to lose if a machine goes down?

Maybe we already talked about this during the previous meeting. If we did, feel free to disregard this comment.